neonvm/tools/vm-builder/files/vector.yaml

```
@@ -15,9 +15,18 @@ sources:
         excludes: ["*/proc/sys/fs/binfmt_misc"]
     type: host_metrics
     scrape_interval_secs: 1 # default is 15, but we scrape every 5s in autoscaler-agent
+  sql_exporter_metrics:
+    type: prometheus_scrape
+    # sql_exporter listen addr, exporting LFC statistics.
+    # This statistics will only be available when using the neon vm image. A custom image
```

```suggestion
    # This statistics will only be available when using the neon compute image. A custom image
```

I thought this was referring to "neonvm" first, which is distinctly separate but closely related :sweat_smile: